internal/storage/driver/memory/binding.go

```diff
@@ -61,3 +61,15 @@ func (s *Binding) ListByInstanceID(instanceID string) ([]internal.Binding, error
 
 	return bindings, nil
 }
+
+func (s *Binding) Get(instanceID string, bindingID string) (*internal.Binding, error) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	binding := s.data[bindingID]
+	if binding.InstanceID == instanceID {
+		return &binding, nil
+	} else {
+		return nil, dberr.NotFound("binding with id %s not exist", bindingID)
```

Comment:
Does not match the `if` condition.
We do not check if bindingID is present in the map, but we check for instanceID equality.

Reply:
And there is function to get binding by bindingID.

Reply:
I want to connect search with both bindingId and instanceId available in URL. Otherwise, it would be possible to load bindings from different instance than the one that instanceId refers to.

Reply:
Got it.

Reply:
Our assumption is that there is one instance for given binding. 
If this assumption is valid we need to enforce it in postgres and in-memory databases.
We do not enforce it neither in in-memory database, nor in postgres database.

This check returns misleading message.
Assume we have in database the pair:
```
binding-1, instance-1
```
and we query for the pair: `binding-1, instance-2`
we get the message: "binding with id binding-1 not exist" - this is obviously not true.